core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5624,6 +5624,14 @@ public String toString() {
               + "other value to use libfuse2 first, libfuse3 if libfuse2 failed")
           .setScope(Scope.ALL)
           .build();
+  public static final PropertyKey FUSE_LIBFUSE3_MAX_IDLE_THREADS =
```

Comment:
We may need a consolidated story about how to set related Fuse configurations.
@ssz1997 can you help understand how many different kinds of config Fuse exposes and see can we have a consolidated way of dealing with each kind of config?

e.g. Fuse mount options
e.g. other Fuse init parameters like MAX_IDLE_THREADS
....
may be others.

Thanks for supporting MAX_IDLE_THREADS. I am a little worried about we keep adding new config like this and have multiple ways of passing fuse config in (especially directly pass the config along the whole path)